jobara/idrc
# chore: update Laravel Mix

## Description

Updates Laravel Mix and package lock file to fix build/lint issues.

## Steps to test

1. `npm ci && npm run lint && npm run build`

**Expected behavior:** Lint and build pass.

## Additional information

Appears that updating Laravel Mix fixed the build error, but the package-lock.json file needed to be recreated to fix the lint error that emerged.

## Related issues

Related to inclusive-design/idrc#166.
